app/controllers/api/user_books_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Api::UserBooksController < ApplicationController
+  before_action :authenticate_user!
+  before_action :get_user_book, only: [:destroy, :update, :show]
+
+  def index
+    books = params[:finish_date] == nil ? UserBook.to_read(current_user) : UserBook.finished(current_user)
+    if books.empty?
+      render json: {errors: "List is empty"}, status: :unprocessable_entity
```

Comment:
I see 3 different error response object structures here:
1. errors: "List is empty" -> singular string
2. errors: user_book.errors -> hash object
3. user_book.errors.full_messages -> array of strings

Do yourself a favor and standardize the structure of your error object. If your API will return different kinds of error response types, then your frontend work will become harder than it should be.